docs/releases.md

```diff
@@ -1,5 +1,97 @@
+
+- **DietPi-Software** | [**Kodi**](../software/media/#kodi)
+    - On **Raspberry Pi Bullseye** systems, the new official **Raspberry Pi** repository build for Kodi 19.3 is now installed. You can apply the upgrade manually by reinstalling Kodi.
+
+        ```sh
+        dietpi-software reinstall 31
+        ```
+
+    - The addon repository is now installed by default with all Kodi installs, which previously was the case only on RPi and Odroids. When currently missing, it can be manually installed
+
+        ```sh
+        apt install kodi-repository-kodi
+        ```
+
+- **DietPi-Software** | **[Gitea](../software/cloud/#gitea)** :octicons-arrow-right-16: The service runs now as dedicated user `gitea` with its home directory `/mnt/dietpi_userdata/gitea` to allow easy transfer and usage of SSH keys for remote access. This applies to newly installed or reinstalled Gitea instances. Many thanks to @LilTrublMakr for reporting the related limitation with the previously used `dietpi` user: <https://github.com/MichaIng/DietPi/issues/4620>.
+
+### Bug Fixes {: #bug-fixes-79 }
+
+- **Raspberry Pi** :octicons-arrow-right-16: Resolved an issue in the DietPi images where on first boot two serial login consoles on the generic `symlinked` and actual serial devices could have been started. This doubled inputs and in turn broke successful `username` and `password` login via serial console on first boot. Many thanks to @ad7718 for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5014>.
+- **DietPi-Software** :octicons-arrow-right-16: Resolved a DietPi v7.8 regression where [ReadyMedia](../software/media/#readymedia), [Deluge](../software/bittorrent/#deluge), [Sonarr](../software/bittorrent/#sonarr) and [Jellyfin](../software/media/#jellyfin) installs failed with an error on `usermod`, since the services were not stopped first. This has been loved via live patches for DietPi v7.8 as well.
+- **DietPi-Software** | **[Transmission](../software/bittorrent/#transmission)**: Resolved a v7.8 regression where on fresh installs the intended configuration was not deployed. Many thanks to @kannz and @alessandro.psrt for reporting these issue on the DietPi forum: ["Transmission settings ?"](https://dietpi.com/phpbb/viewtopic.php?t=9567) or ["Wrong settings.json in transmission-daemon"](https://dietpi.com/phpbb/viewtopic.php?t=9683).
+
+As always, many smaller code performance and stability improvements, visual and spelling fixes have been done, too much to list all of them here. Check out all code changes of this release on GitHub: <https://github.com/MichaIng/DietPi/issues/5019>
+
+### Known/Outstanding Issues {: #known-issues-79 }
```

Comment:
We need to remove that heading from the old release notes then. IMHO it doesn't make sense to keep listing known issues which may have been resolved already, so I remove them from older ones and only keep it for the latest version.